ddtrace/tracer/context_test.go

```diff
@@ -74,3 +74,24 @@ func TestStartSpanFromContext(t *testing.T) {
 	assert.Equal("gin", got.Service)
 	assert.Equal("/", got.Resource)
 }
+
+func TestStartSpanFromNilContext(t *testing.T) {
+	_, _, _, stop := startTestTracer(t)
+	defer stop()
+
+	// Go before 1.15: StartSpanFromContext(nil, ...) worked; calling ctx.Value would segfault
+	// Go == 1.15: StartSpanFromContext(nil, ...) would panic: cannot create context from nil parent
```

Comment:
We can leave these comments out. This commit fixes the panic, so explaining in the comment is unnecessary.